tests/components/dynalite/common.py

```diff
@@ -1,9 +1,69 @@
-"""Common functions for the Dynalite tests."""
+"""Common functions for tests."""
+from asynctest import Mock, call, patch
 
 from homeassistant.components import dynalite
+from homeassistant.setup import async_setup_component
 
+ATTR_SERVICE = "service"
+ATTR_METHOD = "method"
+ATTR_ARGS = "args"
 
-def get_bridge_from_hass(hass_obj):
+
+def create_mock_device(platform, spec):
+    """Create a dynalite mock device for a platform according to a spec."""
+    device = Mock(spec=spec)
+    device.category = platform
+    device.unique_id = "UNIQUE"
+    device.name = "NAME"
+    device.device_class = "Device Class"
+    device.device_info = {
+        "identifiers": {(dynalite.DOMAIN, device.unique_id)},
+        "name": device.name,
+        "manufacturer": "Dynalite",
+    }
+    return device
+
+
+def get_bridge_from_hass(hass):
```

Comment:
We should remove this. We don't want to interact with integration details.

Reply:
i changed it to get the entry_id from hass by asking the config_entries for the dynalite domain. Is this ok?